lib/services/elasticsearch.js

```diff
@@ -251,11 +277,51 @@ function ElasticSearch (kuzzle, options, config) {
     var
       data = cleanData.call(this, requestObject);
 
-    // extends the response with the source from requestObject
-    // When we write in ES, the response from it doesn't contain the initial document content
+    // Add metadata
+    data.body._kuzzle_info = {
+      author: requestObject.userId ? String(requestObject.userId) : null,
```

Comment:
Because if the first time we put an integer we wouldn't be able to put a string next time so I prefer to normalize it with a string